src/connection/manager.ts

```diff
@@ -117,6 +117,7 @@ export class SQLJobManager {
     if (ServerComponent.isInstalled() && selected) {
       return selected.job.query<T>(query, opts);
     } else if(!ServerComponent.isInstalled()) {
+      ServerComponent.checkForUpdate();
```

Comment:
Should the update status be checked here?  If successful, wouldn't we want to avoid throwing the error?

Reply:
Yes. I was thinking the update happens asynchronously and it's reasonable for a user to "try again" after getting the error message. But we should response-check here and make it better

Reply:
https://github.com/halcyon-tech/vscode-db2i/pull/74/commits/dd86d96b4939ffe4d593ec167f249966319bc0ba

Reply:
marking resolved, reopen if needed